### PR TITLE
2046 move docs

### PR DESCRIPTION
**Pull request by ivanmalagon**
Related to #2046 
----

Moved the `guides` and `support` sections from documentation repo to carto.js repo.

### CODE REVIEW

**Comment by ivanmalagon** [comment]
retest this please

**Comment by ivanmalagon** [comment]
retest this please

**Comment by ivanmalagon** [comment]
Let's see. I don't expect those sections to grow too much. If they do we can sum it up a bit more.

File: docs/guides/03-glossary.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Glossary
+
+This glossary defines terms that appear throughout the CARTO.js documentation.
+
+### C
```
Comment (Jesus89) [inline docs/guides/03-glossary.md]: ?

**Pull request merged**